.github/workflows/laravel-tests.yml

```diff
@@ -45,7 +39,115 @@ jobs:
           composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
           composer update --prefer-dist --no-interaction --no-suggest
       - name: Execute tests
-        run: vendor/bin/phpunit
+        run: vendor/bin/phpunit --exclude-group=redis-auth
+        env:
+          REDIS_HOST: redis
+
+  tests-on-php72:
```

Comment:
I assume there's a good reason for this but my first impression is that that's a LOT of redundant configuration for what looks like _could_ be done in a single Matrix using `exclude:` to not test unsupported PHP/laravel versions, similar to https://github.com/rebing/graphql-laravel/blob/ddf9b346a0317729c47254bd4fde9635d5bb3d61/.github/workflows/tests.yml#L8-L21

Apologies if this wasn't appropriate to mention 🙇